App/Redux/TextileNodeRedux.ts

```diff
@@ -147,7 +135,7 @@ export function reducer (state: TextileNodeState = initialState, action: Textile
       const threads = { ...state.threads, [threadId]: { ...threadData, querying: false, error } }
       return { ...state, threads }
     }
-    case getType(actions.refreshMessagesSubmitted):
+    case getType(actions.refreshMessagesRequest):
```

Comment:
No, sorry, this is going the wrong way. Every time there is a Request, you'd toggle refreshing to true. Then in TextileSages, when you go to check if you are already refreshing, you'd always find it to true. I must have somehow dropped the edit, but ~L300 of TextileSagas after `if (!refreshing) {` you actually want to call `TextileNodeActions.refreshMessagesSubmitted` which would only now toggle refreshing to true. Make sense? 

I think it got dropped along the way (100% my fault i'm guessing), I've noticed refresh not working very well since the original branch, now it makes sense

Reply:
Ok, I think this is something we can just solve by constructing the saga in a better way. `takeEvery` and `takeLatest` aren't good for when you want to respond to actions in a serial manner. Improved version of this on the way...